exchanges/bybit/bybit_wrapper.go

```diff
@@ -1653,26 +1653,77 @@ func (e *Exchange) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item)
+				if allInstrumentsInfo.List[x].ContractType == "LinearFutures" ||
+					allInstrumentsInfo.List[x].ContractType == "InverseFutures" {
+					delivery = allInstrumentsInfo.List[x].DeliveryTime.Time()
+				} else {
+					delistedAt = allInstrumentsInfo.List[x].DeliveryTime.Time()
+					// Not entirely accurate but from docs the system will use the average index price in the last
+					// 30 minutes before the delisting time. See: https://www.bybit.com/en/help-center/article/Bybit-Derivatives-Delisting-Mechanism-DDM
+					delistingAt = delistedAt.Add(-30 * time.Minute)
+				}
+			case asset.Spot:
+				// asset.Spot does not return a delivery time and there is no API field for delisting time
+				log.Warnf(log.ExchangeSys, "%s %s: delivery time returned for spot asset", e.Name, pair)
+			}
+		}
+
+		baseStepAmount := allInstrumentsInfo.List[x].LotSizeFilter.QtyStep.Float64()
+		if a == asset.Spot {
+			baseStepAmount = allInstrumentsInfo.List[x].LotSizeFilter.BasePrecision.Float64()
+		}
+
+		maxBaseAmount := allInstrumentsInfo.List[x].LotSizeFilter.MaxOrderQty.Float64()
+		if a != asset.Spot && a != asset.Options {
+			maxBaseAmount = allInstrumentsInfo.List[x].LotSizeFilter.MaxMarketOrderQuantity.Float64()
+		}
+
+		minQuoteAmount := allInstrumentsInfo.List[x].LotSizeFilter.MinOrderAmt.Float64()
+		if a != asset.Spot {
+			minQuoteAmount = allInstrumentsInfo.List[x].LotSizeFilter.MinNotionalValue.Float64()
+		}
+
 		l = append(l, limits.MinMaxLevel{
 			Key:                     key.NewExchangeAssetPair(e.Name, a, pair),
 			MinimumBaseAmount:       allInstrumentsInfo.List[x].LotSizeFilter.MinOrderQty.Float64(),
-			MaximumBaseAmount:       allInstrumentsInfo.List[x].LotSizeFilter.MaxOrderQty.Float64(),
+			MaximumBaseAmount:       maxBaseAmount,
 			MinPrice:                allInstrumentsInfo.List[x].PriceFilter.MinPrice.Float64(),
 			MaxPrice:                allInstrumentsInfo.List[x].PriceFilter.MaxPrice.Float64(),
 			PriceStepIncrementSize:  allInstrumentsInfo.List[x].PriceFilter.TickSize.Float64(),
-			AmountStepIncrementSize: allInstrumentsInfo.List[x].LotSizeFilter.BasePrecision.Float64(),
+			AmountStepIncrementSize: baseStepAmount,
 			QuoteStepIncrementSize:  allInstrumentsInfo.List[x].LotSizeFilter.QuotePrecision.Float64(),
-			MinimumQuoteAmount:      allInstrumentsInfo.List[x].LotSizeFilter.MinOrderQty.Float64() * allInstrumentsInfo.List[x].PriceFilter.MinPrice.Float64(),
-			MaximumQuoteAmount:      allInstrumentsInfo.List[x].LotSizeFilter.MaxOrderQty.Float64() * allInstrumentsInfo.List[x].PriceFilter.MaxPrice.Float64(),
+			MinimumQuoteAmount:      minQuoteAmount,
+			MaximumQuoteAmount:      allInstrumentsInfo.List[x].LotSizeFilter.MaxOrderAmt.Float64(),
+			Delisting:               delistingAt,
+			Delisted:                delistedAt,
+			Expiry:                  delivery,
+			PriceDivisor:            priceDivisor,
+			MultiplierDecimal:       1, // All assets on Bybit are 1x
```

Comment:
```suggestion
			Listed:                         allInstrumentsInfo.List[x].LaunchTime.Time(),
			MultiplierDecimal:       1, // All assets on Bybit are 1x
```
All assets minus spot have the list time, You can do some stuff like "if !time.IsZero()` but it probably all works out the same